app/models/form/sales/pages/organisation.rb

```diff
@@ -11,25 +11,29 @@ def questions
   end
 
   def routed_to?(log, current_user)
-    return false unless current_user
-    return true if current_user.support?
+    if FeatureToggle.merge_organisations_enabled?
```

Comment:
Is it worth using the existing returns or refactoring these conditions to use a similar phrasing. I can see in some instances the feature toggle neatly encloses the method and in one occasion you add it to a condition.
Here it feels messy when followed immediately by return unless. But I can also see the benefit of having a neat encapsulation of the feature flag for the whole method.
I guess I my problem is actually with them lines